docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -95,18 +85,10 @@ KUBE_SERVICE_ADDRESSES="--service-cluster-ip-range=10.254.0.0/16"
 KUBE_API_ARGS=""
 ```
 
-* Edit /etc/etcd/etcd.conf,let the etcd to listen all the ip instead of 127.0.0.1, if not, you will get the error like "connection refused". Note that Fedora 22 uses etcd 2.0, One of the changes in etcd 2.0 is that now uses port 2379 and 2380 (as opposed to etcd 0.46 which userd 4001 and 7001).
-
-```shell
-ETCD_LISTEN_CLIENT_URLS="http://0.0.0.0:4001"
-```
-
-* Create /var/run/kubernetes on master:
+* Edit /etc/etcd/etcd.conf,let the etcd to listen all the ip instead of 127.0.0.1, if not, you will get the error like "connection refused".
```

Comment:
Edit /etc/etcd/etcd.conf to let etcd listen to all available IPs instead of 127.0.0.1; if you haven't done this, you might see an error such as "Connection Refused."